File: Livro - Módulo Java/2 - Estrutura Sequencial/Exercicio.05.java
// 5 - Faça um algoritmo que leia duas notas, calcule e mostre a média ponderada dessas notas, sabendo que os pesos devem ser dados pelo usuário. 

import java.util.Scanner;

public class MediaPonderada {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite a primeira nota: ");
        double nota1 = scanner.nextDouble();

        System.out.print("Digite o peso da primeira nota: ");
        double peso1 = scanner.nextDouble();

        System.out.print("Digite a segunda nota: ");
        double nota2 = scanner.nextDouble();

        System.out.print("Digite o peso da segunda nota: ");
        double peso2 = scanner.nextDouble();

        double mediaPonderada = (nota1 * peso1 + nota2 * peso2) / (peso1 + peso2);

        System.out.println("A média ponderada é: " + mediaPonderada);

        scanner.close();
    }
}

File: Livro - Módulo Python/2 - Estrutura Sequencial/Exercicio.05.py
#5. Faça um algoritmo que leia duas notas, calcule e mostre a média ponderada dessas notas, 
# sabendo que os pesos devem ser dados pelo usuário.

nota1 = float(input('Digite a nota 1:'))
pesoNota1 = float(input('Digite o peso da nota 1:'))
nota2 = float(input('Digite a nota 2:'))
pesoNota2 = float(input('Digite o peso da nota 2:'))

mediaPonderada =  ((nota1 * pesoNota1)+(nota2 * pesoNota2)) / (pesoNota1+pesoNota2)
print(f'Sua nota: {mediaPonderada:.2f}')